_data/nav--top.yml

```diff
@@ -0,0 +1,19 @@
+# This file defines the structure of various named navigational
```

Comment:
One thing that sticks out is that the component is called `nav__top`, but the README says you can use it anywhere. Why not just call it `nav`? Also, is the idea that we can going to have multiple nav templates for different layout configurations? 

Say for example, the user wants 2 navs, one horizontal and another vertical. Are they supposed to just copy this file and change up the CSS, or are we going to provide config options?

Reply:
This is too be used for the top navigation. The code varies a great deal between if the navigation is a top nav or a side nav. 

- Side nav - https://components.standards.usa.gov/components/detail/sidenav--default.html
- Top nav - https://components.standards.usa.gov/components/detail/header--basic.html

I'm thinking there would be config options to support what type of horizontal nav you want or what type of side nav you want.

Reply:
So I guess 'top nav' just means any nav that isn't on the side of a given container element?

Reply:
Yeah, I was thinking in terms of following component breakdown from the USDWS.